R/distanceMatrix.R

```diff
@@ -4,8 +4,8 @@
 #' @param t An optional list of the same size as \code{q} containing evaluation
 #'   grids for each QTS.
 #' @param labels A character vector specifying labels for each QTS.
-#' @param step_pattern The choice for the step pattern of the warping path (see
-#'  \code{\link[dtw]{stepPattern}} for more information)
+#' @param step_pattern The choice for the step pattern of the warping path. (default to \code{dtw::symmetric2},
+#' which is symmetric, normalizable with no local slope constraints. See \code{\link[dtw]{stepPattern}} for more information).
```

Comment:
Please use `@inheritParams DTW` to avoid documenting the same parameters over multiple functions.